tests/wlr_virtual_pointer_v1.cpp

```diff
@@ -95,8 +109,7 @@ class VirtualPointerV1Test: public wlcs::StartedInProcessServer
         EXPECT_CALL(listener, frame()).Times(AnyNumber());
         the_server().move_surface_to(surface, 0, 0);
         pointer.move_to(pointer_start_x, pointer_start_y);
-        send_client.roundtrip();
-        receive_client.roundtrip();
+        wait_for_quiescence();
```

Comment:
This isn't necessary; we really are just waiting for roundtrips here (there's no virtual pointer usage)